### ISSUE TITLE
Add support for jump to line from Roku log output

### ISSUE THREAD

**Issue by triwav** [rokucommunity/vscode-brightscript-language#319]
If we get something like
```
Found 1 compile error
--- Return must return a value. (compile error &ha9) in pkg:/components/.../name.brs(27)
```
then we should to click on the pkg part to jump to that line.

**Comment by chrisdp** [comment]
Right now if we have enough information to find the file we will open it for you on compile error. Also with your example what it a full path or did it actually have the ... in it?

I feel like we do support this now so I'm wondering if it's a format thing.

**Comment by triwav** [comment]
Yeah it did seem like it worked some in the past and yes it's a full path it gave

**Comment by chrisdp** [comment]
This has been addressed via https://github.com/rokucommunity/roku-debug/pull/208
